pkg/config/constants.go

```diff
@@ -166,135 +168,4 @@ var (
-		"Microsoft.Network/networkInterfaces/delete",
-		"Microsoft.Network/networkInterfaces/join/action",
-		"Microsoft.Network/networkInterfaces/read",
-		"Microsoft.Network/networkInterfaces/write",
-		"Microsoft.Network/networkSecurityGroups/read",
-		"Microsoft.Network/networkSecurityGroups/securityRules/delete",
-		"Microsoft.Network/networkSecurityGroups/securityRules/read",
-		"Microsoft.Network/networkSecurityGroups/securityRules/write",
-		"Microsoft.Network/publicIPAddresses/delete",
-		"Microsoft.Network/publicIPAddresses/join/action",
-		"Microsoft.Network/publicIPAddresses/read",
-		"Microsoft.Network/publicIPAddresses/write",
-		"Microsoft.Network/routeTables/delete",
-		"Microsoft.Network/routeTables/read",
-		"Microsoft.Network/routeTables/write",
-		"Microsoft.Resources/subscriptions/resourceGroups/read",
-		"Microsoft.Network/privateDnsZones/read",
-		"Microsoft.Network/privateDnsZones/write",
-		"Microsoft.Network/privateDnsZones/delete",
-		"Microsoft.Network/privateDnsZones/virtualNetworkLinks/read",
-		"Microsoft.Network/privateDnsZones/virtualNetworkLinks/write",
-		"Microsoft.Network/privateDnsZones/virtualNetworkLinks/delete",
-		"Microsoft.Network/virtualNetworks/join/action",
-		"Microsoft.Network/privateDnsZones/A/write",
-		"Microsoft.Network/privateDnsZones/A/read",
-		"Microsoft.Network/privateDnsZones/A/delete",
-		"Microsoft.Storage/storageAccounts/blobServices/containers/write",
-		"Microsoft.Storage/storageAccounts/blobServices/containers/read",
-		"Microsoft.Storage/storageAccounts/write",
-		"Microsoft.Storage/storageAccounts/read",
-		"Microsoft.Storage/storageAccounts/blobServices/listKeys/action",
-		"Microsoft.Network/virtualNetworks/write",
-		"Microsoft.Network/virtualNetworks/read",
-		"Microsoft.Network/virtualNetworks/delete",
-		"Microsoft.Network/virtualNetworks/virtualMachines/read",
-		"Microsoft.Network/virtualNetworks/virtualNetworkPeerings/read",
-		"Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write",
-		"Microsoft.Network/virtualNetworks/virtualNetworkPeerings/delete",
-		"Microsoft.Network/virtualNetworks/peer/action",
-		"Microsoft.Network/virtualNetworks/join/action",
-		"Microsoft.Network/virtualNetworks/joinLoadBalancer/action",
-		"Microsoft.Network/virtualNetworks/subnets/write",
-		"Microsoft.Network/virtualNetworks/subnets/read",
-		"Microsoft.Network/virtualNetworks/subnets/delete",
-		"Microsoft.Network/virtualNetworks/subnets/virtualMachines/read",
-		"Microsoft.Network/virtualNetworks/subnets/join/action",
-		"Microsoft.Network/virtualNetworks/subnets/joinLoadBalancer/action",
-		"Microsoft.Compute/images/write",
-		"Microsoft.Compute/images/read",
-		"Microsoft.Compute/galleries/write",
-		"Microsoft.Compute/galleries/read",
-		"Microsoft.Compute/galleries/images/write",
-		"Microsoft.Compute/galleries/images/read",
-		"Microsoft.Compute/galleries/images/versions/read",
-		"Microsoft.Compute/galleries/images/versions/write",
-	}
```

Comment:
This will remain in the history on GitHub for us to reference as we create docs for Palette later.